.papr.yml

```diff
@@ -18,14 +18,19 @@ cluster:
   container:
     image: registry.fedoraproject.org/fedora:26
 
-packages:
-  - git
-  - rsync
+# https://bugzilla.redhat.com/show_bug.cgi?id=1483553
+#packages:
+#  - git
+#  - rsync
 
 env:
   HOSTS: vmcheck1 vmcheck2 vmcheck3
 
 tests:
+  # hack for https://bugzilla.redhat.com/show_bug.cgi?id=1483553
+  - if (dnf upgrade -y || :) |& grep -q -e BDB1539; then
+    rpm --rebuilddb; dnf upgrade -y; dnf install -y git rsync;
```

Comment:
I tend to always use `yum` so the command is easily portable myself.

Reply:
Also we need to install the packages even if the upgrade failed, right?

Reply:
Done below! ⬇️ This is hopefully very temporary so I was comfortable just hardcoding `dnf`.
... And now debugging why the tests aren't starting. Looks like there's 🔥 everywhere today!

Reply:
The `yum -y install` line needs to be outside the `if`, correct?

Reply:
Yup, good catch, fixed below! ⬇️